app/controllers/scheduled_messages_controller.rb

```diff
@@ -18,7 +18,8 @@ def create
   end
 
   def index
-    @scheduled_messages = ScheduledMessage.filter(params).order(:days_after_start).page(params[:page])
+    @scheduled_messages = ScheduledMessage.filter(params)
```

Comment:
should we be using `date_time_ordering` here? 

also, let's write a feature test for this so we have a broken test if sorting breaks!

consider using an approach like this: http://stackoverflow.com/a/10847756/1019369